src/main/java/org/jabref/logic/importer/fileformat/MarcXmlParser.java

```diff
@@ -109,10 +110,14 @@ private BibEntry parseEntry(Element element) {
                 putPhysicalDescription(bibEntry, datafield);
             } else if ("490".equals(tag) || "830".equals(tag)) {
                 putSeries(bibEntry, datafield);
+            } else if ("502".equals(tag)) {
+                putThesisDescription(bibEntry, datafield); // Master's thesis, PhD thesis, Thesis
             } else if ("520".equals(tag)) {
                 putSummary(bibEntry, datafield);
             } else if ("653".equals(tag)) {
                 putKeywords(bibEntry, datafield);
+            } else if ("773".equals(tag)) {
+                putIssue(bibEntry, datafield);
```

Comment:
For the additions, I don't see any test cases (test files!) added. Do you have some at hand to add to the prepository?

Reply:
No, I haven't thought about tests yet. I was working on the proceedings. I would like to do the test cases next. There is the BvbFetcherTest class where I can put the tests. Do I write two tests for both methods I have implemented?

Reply:
I would not test the inner code of the fetcher too much, but rather the API and the results. if you put some marcxml with various bibtextypes in it returns the expected bibentry. Would require to build some test files in XML and put to the resources directory in the test directory. There should already be some similar tests.

Reply:
Maybe distinguish between the fetcher and the importer. That way you can focus testing the importer if it returns anything usable and test the importer, if he does the parsing right.